### ISSUE TITLE
Parallel Coordinates is getting slower

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#230]
There seems to be a dependency or something that has increased the amount of time that parallel coordinates takes (already a significant amount of time). I ran the tests on my local 2.7 and 3.5 environments and got approximately 24s-1m run times. However on Travis it took > 10m. 

pip freeze:

```
alabaster==0.7.9
Babel==2.3.4
coverage==4.2
cycler==0.10.0
docutils==0.12
funcsigs==1.0.2
imagesize==0.7.1
Jinja2==2.8
MarkupSafe==0.23
matplotlib==1.5.3
mock==2.0.0
nose==1.3.7
numpy==1.11.2
pbr==1.10.0
Pygments==2.1.3
pyparsing==2.1.10
python-dateutil==2.5.3
pytz==2016.7
scikit-learn==0.18
scipy==0.18.1
six==1.10.0
snowballstemmer==1.2.1
Sphinx==1.4.8
sphinx-rtd-theme==0.1.9
```

**Comment by rebeccabilbro** [comment]
Need to test each combination of dependency versions since the last to identify the deprecation warning. Most likely candidates are, in order: numpy, scipy, matplotlib.

**Comment by bbengfort** [comment]
Looks like it's matplotlib 2.0.2 -- I was on matplotlib 1.5.3 locally and travis is installing the latest version of mpl.

**Comment by bbengfort** [comment]
Oh, actually numpy also upgraded ...

**Comment by bbengfort** [comment]
Fixed in #420 and #447
